src/bandersnatch/mirror.py

```diff
@@ -441,7 +443,9 @@ def normalized_legacy_simple_directory() -> Path:
                     f"Attempting to cleanup non PEP 503 simple dir: {deprecated_dir}"
                 )
                 try:
-                    rmtree(deprecated_dir)
+                    for file in deprecated_dir.glob("*"):
+                        file.unlink(missing_ok=True)
+                    deprecated_dir.rmdir()
```

Comment:
I changed rmtree to storage.unlink and rmdir method, but I dont know how to mock it, it seems to be complicated. Can you help me with it?

Reply:
Should just be able to mock `bandersnatch.mirror.Path.unlink` + rmdir ...